tests/foreman/api/test_provisioning.py

```diff
@@ -599,7 +598,7 @@ def test_rhel_pxe_provisioning_fips_enabled(
 @pytest.mark.upgrade
 @pytest.mark.on_premises_provisioning
 @pytest.mark.parametrize('pxe_loader', ['secureboot'], indirect=True)
-@pytest.mark.rhel_ver_match([get_sat_rhel_version().major])
+@pytest.mark.rhel_ver_match('[^6]')
```

Comment:
I think 6 was dropped from supportablity, so I guess you can just do "*"

Reply:
Also, * would select all tests including fips but we should now replace this pattern as it doesn't make sense since we dropped the RHEL6 support.